src/azure-cli/azure/cli/command_modules/appconfig/keyvalue.py

```diff
@@ -67,11 +68,16 @@ def import_config(cmd,
                   src_connection_string=None,
                   src_key=None,
                   src_label=None,
+                  src_snapshot=None,
                   preserve_labels=False,
                   src_auth_mode="key",
                   src_endpoint=None,
                   # from-appservice parameters
                   appservice_account=None):
+
+    if src_snapshot and (src_key or src_label or skip_features):
+        raise CLIErrors.MutuallyExclusiveArgumentError("'--src-snapshot' cannot be specified with '--src-key', '--src-label' filters, or '--skip-features' options.")
```

Comment:
Can we do this validation with the argument validators?

Reply:
That's a good idea. This has been moved. Thanks!